package.json

```diff
@@ -480,7 +476,12 @@
         "titanium.project.defaultI18nLanguage": {
           "type": "string",
           "default": "en",
-          "description": "%titanium.config.project.defaultI18nLanguage%"
+          "description": "Default language to use for i18n autocomplete."
+        },
+        "titanium.ios.organizations": {
+          "type": "string",
+          "default": "",
+          "description": "Input JSON object with your organization's names and IDs. "
```

Comment:
Could we add this into `package.nls.json` and then reference using that ID so that it's setup for localization in future..I think `titanium.config.ios.organizations` should be suitable.

Alongside that could we also document what the expected string should look like?

Reply:
`[
    {
        "id": "TZF9D6738F",//organisation ID from certificate/apple developer/etc
        "name": "1st Organisation" //name to show, can be anything
    },
    {
        "id": "73QRFU8MDC",
        "name": "Second one" 
    },
    
]`
Is this okay?
To type change - I tried it with various types, but it throwed errors, so I stayed with string
I also added entry to package.nls.json

Reply:
If we used `"type": "object'` then we can get a table like below that would accept a key/value data, the certificate ID could be the key and the value could be the custom name. My concern with JSON in a string is that editing isn't really simple and we would have to handle data validation and error reporting in the case of bad input.

<img width="568" alt="image" src="https://github.com/user-attachments/assets/dd3977ae-9254-4858-b722-6f7e92786102">

The table is returned as an object so would change how we access but that shouldn't be an issue.

```
{
 "TZF9D6738F": "1st Organisation"
 "73QRFU8MDC": "Second one"
}
```